slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -316,7 +316,7 @@ std::string SLAMServiceImpl::GetNextDataFileOffline() {
         throw std::runtime_error("no data in data directory");
     }
     if (current_file_offline == file_list_offline.size()) {
-        LOG(INFO) << "Finished processing offline images";
+        LOG(INFO) << "Finished processing offline data";
```

Comment:
I thought it would be more correct to say "data" than "images" for cartographer.

Reply:
[nit] I think it would be helpful to add a comment that clarifies that this log is needed for integration tests, such that someone doesn't accidentally delete it